SEED-platform/seed
# Fix default report X and Y axis selectors

#### What's this PR do?
- Fix default report X and Y dropdowns to be easier to select
- Fixed a strange code duplication that broke the inventory filter
-
#### How should this be manually tested?
Load the default reports page. make sure you can click on the dropdown to open it.
#### What are the relevant tickets?
#4353 

#### Screenshots (if appropriate)
<img width="931" alt="Screenshot 2024-02-23 at 3 09 02 PM" src="https://github.com/SEED-platform/seed/assets/2205659/f7660c06-4d05-47fe-a354-d8cec5a3185e">
